### PR TITLE
Temporarily add CI for hip on ault

### PR DESCRIPTION
**Pull request by aurianer**
Since we cannot test HIP on rostam for several weeks, this temporarily run CI on ault.
This is not aiming to be a long standing CI as ault is not a production machine.
[skip perftests]

### CODE REVIEW

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

File: .jenkins/cscs-ault/batch.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+set -eux
+
+orig_src_dir="$(pwd)"
+hpx_dir="/dev/shm/hpx"
+src_dir="${hpx_dir}/src"
+build_dir="${hpx_dir}/hpx/build"
+install_dir="${hpx_dir}/hpx/install"
```
Comment (msimberg) [inline .jenkins/cscs-ault/batch.sh:15]:
```suggestion
build_dir="${hpx_dir}/build"
install_dir="${hpx_dir}/install"
```
?

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Comment by aurianer** [comment]
retest hip

**Pull request closed without merging**